include/fmt/format.h

```diff
@@ -2699,9 +2699,10 @@ inline void vformat_to(
   return detail::vformat_to(buf, to_string_view(format_str), args);
 }
 
-template <typename S, typename... Args, size_t SIZE = inline_buffer_size,
-          typename Char = enable_if_t<detail::is_string<S>::value, char_t<S>>>
-inline auto format_to(basic_memory_buffer<Char, SIZE>& buf, const S& format_str,
+template <typename S, typename... Args, typename Char, size_t SIZE,
+          typename Allocator, FMT_ENABLE_IF(detail::is_string<S>::value)>
+inline auto format_to(basic_memory_buffer<Char, SIZE, Allocator>& buf,
+                      const S& format_str,
                       Args&&... args) ->
```

Comment:
Does clang-format still break this onto the next line?

Reply:
Oops, thanks, this time run through clang-format.